Neos.Flow/Classes/Configuration/ConfigurationManager.php

```diff
@@ -423,6 +423,11 @@ protected function processConfigurationType(string $configurationType)
             $this->writeConfigurationCacheFile($cachePathAndFilename, $configurationType);
             $this->replaceConfigurationForConfigurationType($configurationType, $cachePathAndFilename);
             @unlink($cachePathAndFilename);
+        } else {
+            // in case the cache is disabled (implicitly, because temporaryDirectoryPath is null)
```

Comment:
This was in fact never a planned use case ;)

Reply:
```
    /**
     * An absolute file path to store configuration caches in. If null no cache will be active.
     *
     * @var string
     */
    protected $temporaryDirectoryPath;
```

this has been there since EVER!

Reply:
you build it ^^ more or less accidental: https://github.com/neos/flow-development-collection/commit/85a7f15664cca6dfba7911b1ed8ee34cc2296102

Reply:
and later the feature Runtime evaluation of env and constants in Configuration was added https://github.com/neos/flow-development-collection/commit/1ca7951743ac780bfb6edb83435532eafabd2ef3#diff-83398c2914c2ce46b6e55c9821468814652a275d332692492bb6edbbb84607f9R617

Reply:
Yep, I know, and I _think_ there was some low level use case at that time that required it to work that way. I would rather close that road than open it :D